atlascope/core/models/dataset.py

```diff
@@ -5,14 +5,18 @@
 from guardian.admin import GuardedModelAdmin
 from rest_framework import serializers
 
+from .importer import importers
+
+available_importer_choices = [(importer_name, importer_name) for importer_name in importers]
```

Comment:
Yes, this is a Django thing: https://docs.djangoproject.com/en/4.0/ref/models/fields/#:~:text=A%20sequence%20consisting,name.%20For%20example%3A

Reply:
Ah I see. We will probably want to use the "human readable name" aspect of this in the future. Could you file an issue to divorce the importer name from a human-readable name?

Reply:
Not sure what you mean by divorce; any time we access the field, we will get the human-readable version by default. The "actual value" as Django calls it is simply what is saved in the db column. So If you have really long human-readable values that can be simplified to two-character codes, then there is no need to waste db space and you can set the max length of the field to 2 while the values you get back are much longer.

I'm not sure we can do much to avoid this structure for what `CharField` expects of the `choices` array...

Reply:
All I meant was, you're currently duplicating a single name into both values, and we might want to vary the names in the future. But TBH, this is not important enough to do anything further with at the moment, so I'll just resolve this thread. Thanks!